x/ccv/provider/types/msg_test.go

```diff
@@ -647,3 +647,57 @@ func TestMsgAssignConsumerKeyValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateInitialHeight(t *testing.T) {
+	testCases := []struct {
+		name          string
+		chainId       string
+		initialHeight clienttypes.Height
+		expPass       bool
+	}{
+		{
+			name:    "valid with revision number",
```

Comment:
Please consider adding a testcase and checking if the code works for evmos-like chain names:
* [evmos_9001-2](https://github.com/cosmos/chain-registry/blob/master/evmos/chain.json#L9)